### PR TITLE
Add automated reminder for API updates

### PR DESCRIPTION
**Pull request by andyw8**
### WHY are these changes introduced?

Fixes #0000 

### Checklist

- [ ] I have updated the CHANGELOG (if applicable)
- [ ] I have followed the [Shopify Python](https://github.com/Shopify/shopify_python) guide

### CODE REVIEW

File: .github/workflows/api_update_reminder.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+on:
+  - workflow_dispatch # to allow for manual testing
+  - schedule:
+      cron: "0 0 1 3,6,9,12" # At 00:00 on 1st of March, June, September, and December
```
Comment (paulomarg) [inline .github/workflows/api_update_reminder.yml:4]: I wonder if a month lead time might be a little too long. Should we change this to trigger on the 15th of those months?

Comment (andyw8) [reply .github/workflows/api_update_reminder.yml]: I padded as I thought testing for the new release may uncover things that need to be updated/fixed.

Comment (paulomarg) [reply .github/workflows/api_update_reminder.yml]: Makes sense, my main concern was whether there was a risk of things being added during this period, after we make our changes.

File: .github/api_update_reminder.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The release schedule can be found at https://shopify.dev/concepts/about-apis/versioning
+
+Review the changelog and consider if anything in the library needs to change:
+
+https://shopify.dev/changelog
+
+Test against the upcoming release by using the Release Candidate.
+
+Another reminder on the created on the date of the next release.
```
Comment (paulomarg) [inline .github/api_update_reminder.md]:
```suggestion
Another reminder will be sent when the Release Candidate is released.
```

Comment (thecodepixi) [inline .github/api_update_reminder.md]: Slight typo here

File: .github/api_update_reminder.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+The release schedule can be found at https://shopify.dev/concepts/about-apis/versioning
+
+Review the changelog and consider if anything in the library needs to change:
+
+https://shopify.dev/changelog
+
+Test against the upcoming release by using the Release Candidate.
+
+Another reminder issue will be created on the date of the next release.
```
Comment (paulomarg) [inline .github/api_update_reminder.md:23]: Is there any chance we can add a comment to the same issue?

File: .github/api_update_reminder.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+When that happens, test again using the Stable API version, and aim to release an update of the library within one week.
```
Comment (paulomarg) [inline .github/api_update_reminder.md:24]:
```suggestion
When that happens, test again using the now Stable API version, and aim to release an update of the library within one week.
```

**Review by thecodepixi — APPROVED** [review]
LGTM 👍

File: .github/api_update_reminder.md (hunk 4 above)
Comment (thecodepixi) [reply .github/api_update_reminder.md:24]: +1

File: .github/api_update_reminder.md (hunk 3 above)
Comment (andyw8) [reply .github/api_update_reminder.md:23]: In theory, yes, but I think it might be a bit tricky to script...

Comment (paulomarg) [reply .github/api_update_reminder.md:23]: Nothing wrong with using a second issue then!

File: .github/api_update_reminder.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: A new release of the Shopify API is due soon
+assignees: andyw8 # @platform/dev-tools-education
```
Comment (andyw8) [inline .github/api_update_reminder.md:3]: Will be changed to team once finalized.

**Review by mllemango — APPROVED** [review]

**Comment by andyw8** [comment]
I added the reminder to go out on day of release for the API.

**Review by paulomarg — APPROVED** [review]
LGTM!

**Comment by andyw8** [comment]
This failing, looking into: https://github.com/Shopify/shopify_python_api/actions/runs/707075633

**Pull request merged**